### DISCUSSION TITLE
Please help me find what's wrong with my config

### DISCUSSION THREAD

**Discussion by augustocdias** [nvim-lualine/lualine.nvim#495] [Q&A] [answered]
### Self Checks

- [x ] I'm using the latest lualine.
- [ x] I didn't find the issue in exsisting issues or prs.

### How to reproduce the problem
When I move around a buffer the progress goes over 100%. If I go to another window and come back, it fixes to the correct value, but as soon as I move the cursor, it goes over again.

<img width="842" alt="image" src="https://user-images.githubusercontent.com/1080789/146520996-0fe28e73-20cf-4ad3-be6c-90aee52b06b6.png">

I don't know what's wrong with my config... I thought it was a bug but when I setup a minimal config it didn't happen. Where can I look to find out what's wrong? I suspect it is the formatting that it is not cleaning the whole section and rewriting it, and when there's movement because of the progress or the location changes the length (or diagnostics). 

This is also happening with the location btw...

**Comment by shadmansaleh** [comment, marked as answer]
See if disabling `lsp-status` fixes it. I think I had another report earlier where lsp-status was causing similar issue.

**Reply by augustocdias** [reply]
That works. I'll open an issue on their repo. Thanks
